RecoHGCal/TICL/plugins/HGCGraph.cc

```diff
@@ -65,7 +65,7 @@ void HGCGraph::makeAndConnectDoublets(const TICLLayerTiles &histo,
               const auto etaRangeMin = std::max(0, oeta - deltaIEta);
               const auto etaRangeMax = std::min(oeta + deltaIEta, nEtaBins);
 
-              for (int ieta = etaRangeMin; ieta < etaRangeMax; ++ieta) {
+              for (int ieta = etaRangeMin; ieta < etaRangeMax + 1; ++ieta) {
```

Comment:
is this update correct?
- a similar ```+1``` is not present in the outer loop on L57
- in a case that ```etaRangeMax``` is equal to ```nEtaBins``` there will be out of bound reads on L77 (looking at array index ```nEtaBins*nPhiBins + iphi```

Reply:
Hi @slava77 thank you. It should be fixed now.